python/pfs/drp/stella/reduceArcRefSpecTask.py

```diff
@@ -42,6 +44,8 @@ def __call__(self, args):
                 result = task.run(**args)
             except Exception, e:
                 task.log.fatal("Failed: %s" % e)
+                traceback.print_exc(file=sys.stderr)
```

Comment:
Why not just throw the exception?  Can't the user can use `--doraise` if desired?